docs/extend/update-1_x.md

```diff
@@ -6,6 +6,62 @@ If you need help applying these changes or using new features, please start a di
 
 :::
 
+## 1.2 Changes
+
+Flarum v1.2 included quite a few bugfixes, internal refactors, and new features. The following recaps the most important changes for extension developers:
+
+### Frontend
+
+- Flarum core now passes TypeScript type checking (on the portion written in TypeScript). Additionally, major portions of the frontend (models, the application instance, and others) are now written in TypeScript. These changes should make it much easier and more fruitful to write extensions in TypeScript.
+- Instead of directly using LESS variables in CSS code, core now uses CSS variables. For the most part, we've just created CSS variables and set their values to the LESS variables. This should make theming and customizing CSS a lot easier. https://github.com/flarum/core/pull/3146.
```

Comment:
```suggestion
- Instead of directly using Less variables in CSS code, core now uses CSS variables. For the most part, we've just created CSS variables and set their values to the Less variables. This should make theming and customizing CSS a lot easier. https://github.com/flarum/core/pull/3146.
```